### DISCUSSION TITLE
Carona compartilhada para o BrazilJS saindo de SP

### DISCUSSION THREAD

**Discussion by RatoX** [frontendbr/forum#1196] [Dicas]
**N**egads,

Por motivos de agenda eu e @lucasmaiaesilva acabamos comprando os ingressos para o BrazilJS bem tarde e as passagens acabaram ficando bem cara, resolvemos alugar um carro e fazer uma _js road trip_, estamos procurando uma terceira pessoa que queira fazer parte dessa zuera.

Saíremos de São Paulo - SP dia 23 ( quinta-feira ) as 8 da manhã, com previsão de chegada no mesmo dia a noite e voltaremos no domingo pela manhã com previsão de chegada no domingo a noite.

Faremos a trip sem pressa e curtindo, nos dois dirigimos e eu tenho uma boa experiencia com estradas já que sou motociclista a 8 anos e já cruzei alguns estados no lombo de uma motoca 250cc.

Enfim, vai ser duka!!

**Comment by felipefialho** [comment]
Vou divulgar a issue 😄

**Comment by giovannism20** [comment]
Queria muito ir... :/

**Comment by lucasdaiki** [comment]
Ainda tem vaga, @RatoX ?

**Comment by annacruz** [comment]
O Rato acabou indo de avião, mas mais alguém topa esse rolê?
